oat-sa/extension-tao-itemqti
# fix: delete outcomes if no responses set

Ticket: https://oat-sa.atlassian.net/browse/AUT-3735

Fixed the issue when after deleting of an interaction which was contained responses and custom outcome declarations, they will be not deleted properly and will generate inconsistency inside XML

## What's Changed

- Custom outcome declarations are deleted if there are no responses in the item

## How to test
- Create an item with choice interaction, for example, and create a Human response declaration and save it
- Put in the item some shared stimulus with a file and delete the previously added choice interaction and press save
- All outcome declarations should be deleted if no more interactions with responses

Appropriate PR in the npm package: https://github.com/oat-sa/tao-item-runner-qti-fe/pull/409
**Deployed** to https://construct1-oat-unit07.dev.gcp-eu.taocloud.org/tao/Main/login
